octobot_trading/channels/exchange_channel.py

```diff
@@ -81,16 +81,6 @@ async def new_consumer(self,
         await self._check_producers_state()
```

Comment:
Default version is doing the job here ?

Reply:
actually it's the same code, so yes I guess :)

Reply:
Exactly :wink:

Reply:
It was necessary when the comment was written because it was `__check_producers_state` and now it's `_check_producers_state`.